app/controllers/arts_controller.rb

```diff
@@ -0,0 +1,30 @@
+class ArtsController < ApplicationController
+  def new
+    @art = Art.new
+  end
+
+  def show
+    @art = Art.find(params[:id])
+  end
+
+  def create
+    @art = Art.new(art_params)
+    if @art.save
+      respond_to { |format| format.html { redirect_to @art } }
+    else
+      flash[:notice] = "You must fill out all of the required fields"
+      render :new
+    end
+  end
+
+  private
+
+  def art_params
+    params.require(:art).permit(
+    :name, :location,
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Indent the first parameter one step more than the start of the previous line.